form.js

```diff
@@ -1,29 +1,47 @@
-function showMessage(message) {
-  const msg = document.querySelector('small');
-  msg.innerText = message;
-}
-function showError(message) {
-  showMessage(message);
-}
-function showSuccess() {
-  showMessage('');
+const toggle = document.querySelector('.toggle');
+const body = document.querySelector('body');
+const menuOptions = document.querySelectorAll('.menu li');
+
+toggle.addEventListener('click', () => {
+  body.classList.toggle('open');
+});
+
+menuOptions.forEach((option) => {
+  option.addEventListener('click', () => {
+    body.classList.remove('open');
+  });
+});
+//    ----------------------------Storage----------------------------------------------
+
+const nameInput = document.querySelector('.inp-name');
+const emailInput = document.querySelector('.inp-email');
+const messageInput = document.querySelector('.inp-text');
```

Comment:
- [ ] I have noticed that you are selecting the class `.inp-name`, `.inp-email`, and `.inp-text` which do not exist in your HTML file. Kindly ensure to make the correct selection. This is hindering the data from being preserved in the local storage such that when the page is reloaded, the user does not need to enter data again. Kindly fix this to increase to user experience ☺️.

Output
![image](https://user-images.githubusercontent.com/99394374/226071986-aab75470-b1d7-4dcd-ba2c-4f0f27600bb5.png)

![image](https://user-images.githubusercontent.com/99394374/226072236-004374c9-fd23-41ed-bcda-9bfc050c32de.png)